.ci/maven/nexus-settings.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
Do we still need this file?

Reply:
Removed it. I added it for testing purposes and forgot to remove it.